templates/registration/email_change_notify_email.html

```diff
@@ -0,0 +1,24 @@
+<div style="border:2px solid #fd0;margin:4px 0;"><div style="background:#000;border:6px solid #000;">
+<a href="{{ protocol }}://{{ domain }}"><img src="{{ protocol }}://{{ domain }}/static/icons/logo.svg" alt="{{ site_name }}" width="160" height="44"></a>
+</div></div>
+
+<div style="border:2px solid #337ab7;margin:4px 0;"><div style="background:#fafafa;border:12px solid #fafafa;font-family:segoe ui,lucida grande,Arial,sans-serif;font-size:14px;">
+<br><br>
+{{ user.get_username() }},
+<br>
+{% trans %}You have requested to change your email address to {{ new_email }} for your user account at {{ site_name }}.{% endtrans %}
+<br><br>
+{{ _('If this was you, no further action is required.') }}
+
+<br>
+<b>
+{% if site_admin_email %}
+{% with link='<a href="mailto:%(email)s">%(email)s</a>'|safe|format(email=site_admin_email) %}
+{{ _('If this was not you, please email us immediately at %(email)s.', email=link) }}
+{% endwith %}
+{% else %}
+{{ _('If this was not you, please reply to this email immediately.') }}
```

Comment:
Shouldnt we suggest changing pwd as well?

Reply:
Good idea, updated.